### PR TITLE
Fix set identity on accounts

### PR DESCRIPTION
**Pull request by TarikGul**
closes: https://github.com/polkadot-js/apps/issues/10580

### CODE REVIEW

File: packages/page-accounts/src/modals/IdentityMain.tsx [hunk 1]
```diff
@@ -100,8 +103,8 @@ function IdentityMain ({ address, className = '', onClose }: Props): React.React
 
       setData(info.web, setHasWeb, setValWeb);
 
-      [info.display, info.email, info.legal, info.riot, info.twitter, infoDiscord, info.web].some((info: Data) => {
-        if (info.isRaw) {
+      [info.display, info.email, info.legal, info.riot, info.twitter, infoDiscord, info.web].some((info: Data | null) => {
```
Comment (joepetrowski) [inline packages/page-accounts/src/modals/IdentityMain.tsx:106]: See new `IdentityInfo`: https://github.com/polkadot-fellows/runtimes/blob/v1.2.1/system-parachains/people/people-kusama/src/people.rs#L96-L143

- `riot` is now `matrix`
- `github` and `discord` are first class members
- `additional` is no more

```suggestion
      [info.display, info.email, info.legal, info.matrix, info.twitter, info.discord, info.github, info.web].some((info: Data | null) => {
```

Comment (joepetrowski) [reply packages/page-accounts/src/modals/IdentityMain.tsx:106]: Anyone with `additional` fields of `github` or `discord` had those fields migrated into the new struct.

**Comment by joepetrowski** [comment]
Looks like this fixes setting Identity, but there also needs to be a PR to change Identity rendering? As in, when I'm on the Relay Chain, accounts with Identity on the People Chain should still render with that Identity.

File: packages/page-accounts/src/modals/IdentityMain.tsx (hunk 1 above)
Comment (TarikGul) [reply packages/page-accounts/src/modals/IdentityMain.tsx:106]: Ahh thanks for linking that, I will add those additions - Also need to keep the old structures though to make sure other chains don't have this break in the process.

**Comment by TarikGul** [comment]
> Looks like this fixes setting Identity, but there also needs to be a PR to change Identity rendering? As in, when I'm on the Relay Chain, accounts with Identity on the People Chain should still render with that Identity.

Yea this only fixes setting the identity. The actual retrieving of the information is going to be a way larger overhaul. I don't know exactly how I am going to do it yet, but I will have a plan figured out today. Same as the above, but basically we need to keep the old while bringing in the new, so that we don't break everything for every chain that wont be using this identity structure.

**Comment by joepetrowski** [comment]
> The actual retrieving of the information is going to be a way larger overhaul. I don't know exactly how I am going to do it yet

Hmm, I would not expect that, as the Identity on system chains rendered from the Relay Chain. I would expect that it just means always making a wss connection with People Chain and changing the provider of Identity info?

**Comment by TarikGul** [comment]
Tested on both Polkadot, and westend-people and the logic works as it should.

**Pull request merged**